HW1/P5/P5_connected_components.py

```diff
@@ -0,0 +1,62 @@
+import pyspark
+sc = pyspark.SparkContext()
+sc.setLogLevel("ERROR")
+
+def link_string_to_KV(s):
```

Comment:
was this function provided. the other person I am reviewing has also used the exact same one